src/Modules/UngradedModules/FreeModuleHom.jl

```diff
@@ -427,16 +425,10 @@ julia> a = hom(F, G, V);
 
 julia> kernel(a)
 (Submodule with 1 generator
-1 -> x*z*e[1] - y*z*e[2] + y^2*e[3]
-represented as subquotient with no relations., Map with following data
-Domain:
-=======
-Submodule with 1 generator
-1 -> x*z*e[1] - y*z*e[2] + y^2*e[3]
-represented as subquotient with no relations.
-Codomain:
-=========
-Free module of rank 3 over R)
+  1: x*z*e[1] - y*z*e[2] + y^2*e[3]
+represented as subquotient with no relations, Hom: submodule with 1 generator
```

Comment:
Why is Hom written with capitel H? It is not meant to be the module Hom, it is meant to be a homomorphisms.

Reply:
this is to be consistent with all other homomorphisms in OSCAR

Reply:
We just discussed this here. To summarize: the `,` comes from Julia printing a tuple. The second tuple element is a homomorphism. And by our conventions, when we print objects then we capitalize "at the beginning" as if it was a sentence. Of course that's a bit jarring here, because `Hom: ` is not at the start of a line, but rather comes after a comma. But I see no good way to overcome this.

Note that there are plenty of existing doctests that look just like this, e.g.:
```
julia> reduced_scheme(X)
(scheme(x^2 - 2*x*y + y^2, x - y), Hom: scheme(x^2 - 2*x*y + y^2, x - y) -> scheme(x^2 - 2*x*y + y^2))
```
or from invariant theory
```
julia> A, AtoS = affine_algebra(RG)
(Quotient of multivariate polynomial ring by ideal (0), Hom: A -> S)
```

The example here is perhaps worse in so far as the context suggest that one might be talking about a Hom-module. But then I hope that the notation `Hom: submodule ...` eventually makes it clear enough.

Reply:
Yes, sure, we have to make a choice and use this choice as an abbreviation for homomorphism  wherever appropriate. But again, using Hom as abbreviation is confusing. Anyway, I give up at this point. Otherwise, everything looks much better as before. Thx, @fingolfin.

Reply:
I am happy to have `Hom:` changed to something else (but what?). But IMHO it should be done for all (homo)morphisms, not just module homomorphisms. I.e. I think this should be a separate discussion.